certified-connectors/U.S. Bank Treasury Management/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2778 @@
+          "type": "string",
+          "description": "Debtor Name",
+          "example": "WESTERN GLAZIERS RETIREMENT PLANDEP OSIT ACCOUNT"
+        },
+        "debtorAccount": {
+          "type": "string",
+          "description": "Debtor Account",
+          "example": 169709692134
+        },
+        "debtorBankID": {
+          "type": "string",
+          "description": "Debtor Bank Identifier",
+          "example": 123000220
+        },
+        "debtorBank": {
+          "type": "string",
+          "description": "Debtor Bank",
+          "example": "US Bank"
+        },
+        "debtorAddress": {
+          "type": "string",
+          "description": "Debtor Address",
+          "example": "PO BOX 4148 PORTLAND 972084148"
+        },
+        "creditorName": {
+          "type": "string",
+          "description": "Creditor Name",
+          "example": "IBEW & United Workers FCU"
+        },
+        "creditorAccount": {
+          "type": "string",
+          "description": "Creditor Account",
+          "example": 323075194
+        },
+        "creditorBankID": {
+          "type": "string",
+          "description": "Creditor Bank Identifer",
+          "example": 244084264
+        },
+        "creditorBank": {
+          "type": "string",
+          "description": "Creditor Bank",
+          "example": "Corporate One Federal Credit Union"
+        },
+        "creditorAddress": {
+          "type": "string",
+          "description": "Creditor Address",
+          "example": "9955 SE Washington St Portland 97216"
+        }
+      }
+    },
+    "Zelle": {
+      "type": "object",
+      "description": "The avilability of the fields in the below object is dependent on type of transaction.",
+      "properties": {
+        "receiverName": {
+          "type": "string",
+          "description": "Receiver Name",
+          "example": "REBECCA S"
+        },
+        "alias": {
+          "type": "string",
+          "description": "Alias",
+          "example": "aa@usbank.com"
+        },
+        "paymentType": {
+          "type": "string",
+          "description": "Payment Type",
+          "example": "INSTANT"
+        },
+        "effectiveDate": {
+          "type": "string",
+          "format": "date",
+          "description": "Effective Date",
+          "example": 10422
+        },
+        "paymentInstructionID": {
+          "type": "string",
+          "description": "paymentInstructionID",
+          "example": "PI0000001155129"
+        },
+        "parameterName": {
+          "type": "string",
+          "description": "Parameter Name",
+          "example": "record number"
+        },
+        "parameterValue": {
+          "type": "string",
+          "description": "Parameter Value",
+          "example": "abc123"
+        }
+      }
+    },
+    "Wires": {
+      "type": "object",
+      "description": "The avilability of the fields in the below object is dependent on type of transaction.",

```

Comment:
@jzworkman all files should be free of grammatical errors and typos. in the description you have "avilability", please change to "availability".